Answers.md

```diff
@@ -1,5 +1,26 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+
+    - Server side only requests the data that is needed and has better seo optimization. every request results in a full page refresh.
+       Client side loads all the data initially thus granting a slower init load time but fast traversal through pages and data without refreshing.
+        seo is less optimized and it requires more code to get setup because server side is the standard.
+
```

Comment:
Server side loads each page as separate HTML request and reloads for each page change.

Client side loads HTML initially, then updates the page using javascript. Additional updates are performed with AJAX requests and javascript DOM updates rather than full reloads.